backend/src/app.ts

```diff
@@ -1,74 +1,16 @@
 import cors from 'cors';
-import dotenv from 'dotenv';
 import express from 'express';
-import session from 'express-session';
-import memoryStoreFactory from 'memorystore';
-import { fileURLToPath } from 'node:url';
 
-import { importMetaUrl } from './importMetaUtils';
-import { ChatModel, defaultChatModel } from './models/chat';
-import { LevelState, getInitialLevelStates } from './models/level';
-import { router } from './router';
+import nonSessionRoutes from './nonSessionRoutes';
+import sessionRoutes from './sessionRoutes';
 
-dotenv.config();
-
-declare module 'express-session' {
```

Comment:
Much of this file has been moved to sessionRoutes.ts, the rest in nonSessionRoutes.ts